index.test.js

```diff
@@ -0,0 +1,7 @@
+// const foo = require('./src/index.js');
```

Comment:
It is not a good practice not to remove old codes. Kindly remove commented codes 😀